packages/rulesets/src/spectral/functions/tags-are-top-level-properties-only.ts

```diff
@@ -0,0 +1,22 @@
+import _ from "lodash"
+import { deepFindObjectKeyPath, getProperties } from "./utils"
+
+const TAGS = "tags"
+const PROPERTIES = "properties"
+const ERROR_MESSAGE = "Tags must be defined as a top-level property, not in the properties bag."
+
```

Comment:
"Tags should not be specified in the properties bag for proxy resources. Consider using a Tracked resource instead."